### PR TITLE
add project dependencies

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+# Install memory_profiler if this script is run with PROFILE_MEMORY flag
+ARG INSTALL_MEMORY_PROFILER="false"
+RUN if [ "$INSTALL_MEMORY_PROFILER" = "true" ]; then \
+        apt-get update && apt-get install -y gcc && \
+        pip install memory_profiler; \
+    fi
+
+# Install plotly depedencies (orca and gcc)
+ARG ORCA_VERSION="1.2.1"
```
Comment (as2388) [inline Dockerfile]: Drive-by comment: I don't think we need this anymore, as we removed the graphs we were producing with plotly.

From the discussion today it sounded like revisiting graphs was low priority, and we might not use plotly/orca next time anyway, so maybe we can just remove

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+[packages]
+CoreDataModules = {editable = true,git = "https://www.github.com/AfricasVoices/CoreDataModules",ref = "v0.15.1"}
+PipelineInfrastructure = {editable = true,git = "https://www.github.com/AfricasVoices/Pipeline-Infrastructure",ref = "v0.0.12"}
+RapidProTools = {editable = true,git = "https://www.github.com/AfricasVoices/RapidProTools",ref = "v0.3.8"}
+pytz = "*"
+python-dateutil = ">=2.8.0"  # Need this minimum so that isoparse handles T24:00:00 correctly.
+google-cloud-storage = "*"
+pyinstrument = "*"
+plotly = "*"
+requests = "*"  # Needed by plotly's image exporter.
```
Comment (as2388) [inline Pipfile:14]: same here

**Review by as2388 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (IsaackMwenda) [reply Dockerfile]: I agree, have removed this.
